src/components/panels/TimePanel.vue

```diff
@@ -0,0 +1,72 @@
+<template>
+  <b-col md="6">
+            <h3 class="text-center">{{headertext}}</h3>
+    <b-list-group style="width: 100%">
+      <b-list-group-item :style="styleObject"  class="list-group-item-borderless">
+        <span class="pull-left">
+         <b>Currently it is:</b>
+        </span>
+          <b-badge 
+              class="pull-right"
+              style="margin-right: 5px"
```

Comment:
Can we avoid inline styles completely?  I know the old way uses them, but in an overhaul I'd like to get rid of them.

Reply:
yeah, that's probably more for me than new on this... which is why i wanted just to review his additions so the rest of the code can get reviewed as we're going on the base branch